tools/ci_build/build.py

```diff
@@ -260,53 +257,9 @@ def check_md5(filename, expected_md5):
         return False
     return True
 
-#the last part of src_url should be unique, across all the builds
-def download_test_data(build_dir, src_url, expected_md5, azure_sas_key):
```

Comment:
Is this not useful to a developer who needs to get the test data?

Reply:
The developer need more than 50gb disk space for doing that, so far I didn't see anyone that want to try it locally.

Reply:
I have them locally but can't remember if I used this or tools\ci_build\github\download_test_data.py

I found the latter easily enough just now so I'm fine with this being removed from build.py

---
In reply to: [348787462](https://github.com/microsoft/onnxruntime/pull/2446#discussion_r348787462) [](ancestors = 348787462)